build.gradle

```diff
@@ -109,12 +110,48 @@ task sourcesJar(type: Jar) {
     from sourceSets.main.allJava
 }
 
+task javadocJar(type: Jar) {
```

Comment:
remove this task - javadoc task will fail currently. we can add this once we clean up the javadocs and have reviewed the output.  ditto with references to javadoc artifacts in the rest of this file.

Reply:
Ok. I will remove `javadocJar` task